src/dashboard/src/components/administration/urls.py

```diff
@@ -28,6 +28,8 @@
     (r'dips/contentdm/$', 'contentdm_dips'),
     (r'sources/$', 'sources'),
     (r'storage/$', 'storage'),
+    (r'usage/$', 'usage'),
+    (r'usage/clear/(?P<dir_id>\w+)/$', 'usage_clear'),
```

Comment:
Should this be in api/ instead of administration?  Do we care about login-required vs api-key?